test_conformance/math_brute_force/binary_i_double.cpp

```diff
@@ -464,7 +463,6 @@ cl_int Test(cl_uint job_id, cl_uint thread_id, void *data)
 
     if ((error = clFlush(tinfo->tQueue))) vlog("clFlush 3 failed\n");
```

Comment:
do we need to `return error;` in this if now? We'd now return CL_SUCCESS when the function returns even if this error was logged, where as before we'd return the error from the function (I think)

Reply:
Thanks, good catch!

Reply:
On a closer look, it looks like other `clFlush` calls are not fatal; see for example the lines that print "clFlush failed" and "clFlush 2 failed" in this file: the error variable is overwritten in subsequent API calls.  The lack of a `goto exit` here makes me suspect that it was intended for this 3rd `clFlush` call to be non-fatal too.

If you agree that returning `CL_SUCCESS` while "clFlush 3 failed" is acceptable, I'll update the commit message to mention this functional change (and drop the NFC tag).

Reply:
Interesting, thanks for looking into it. Seems like this behaviour affects other math_brute_force tests too and not just this file, but after this MR the behaviour will be consistent everywhere in the math_brute_force_tests of `clFlush` being non-fatal. I think the non-fatal behaviour odd, but that's a separate discussion from this useful clean-up PR.

Your proposal for updating the commit and progressing with this change sounds good to me.

Reply:
Thanks! I've updated the commit message and PR description.